packaging/installer/kickstart-static64.sh

```diff
@@ -189,7 +189,7 @@ done
 
 # ---------------------------------------------------------------------------------------------------------------------
 TMPDIR=$(create_tmp_directory)
-cd "${TMPDIR}" || :
+cd "${TMPDIR}" || TMPDIR="/tmp"
```

Comment:
I don't understand this. Either `cd` or just set the variable? Doesn't sound right.

Reply:
Now that you mention it, i don't even know why we need to enter the temp directory.
We always refer the file with the full path

Reply:
i would probably only consider it as a dummy validation step for the directory creation